src/index.ts

```diff
@@ -1,6 +1,16 @@
 import { env } from "./env";
 import { GatewayIntentBits } from "discord.js";
 import { CustomClient } from "./client";
+import * as Sentry from "@sentry/node";
+
+Sentry.init({
+  dsn: env.SENTRY_SECRET,
+
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0,
+});
 
 // Create a new client instance
 const client = new CustomClient({
```

Comment:
I think the client also has an `"error"` event we can listen for and sent events to Sentry

Reply:
Im not aware of this in the code, where is this exactly ?

Reply:
Got it. I located the event in context.

Reply:
I think it's like this:

```js
client.on('error', err => { ... })
```

https://discord.js.org/#/docs/discord.js/main/class/Client?scrollTo=e-error

Reply:
https://github.com/chainpatrol/discord-bot/pull/24/commits/5c169902515660c3400b49b402141299940194d3